fail2ban/server/action.py

```diff
@@ -532,7 +532,7 @@ def _processCmd(self, cmd, aInfo = None):
 		return self.executeCmd(realCmd, self.timeout)
 
 	@staticmethod
-	def executeCmd(realCmd, timeout=60):
+	def executeCmd(realCmd, timeout=60, testCmd=False):
```

Comment:
Well, understood. Then me confused name of the parameters (why this parameter called testCmd?).
I will make a new branch above your one. Wait a little bit.